homework/Vasili_Zasinets/hw1/CarDemo.java

```diff
@@ -0,0 +1,69 @@
+package com.tutrit.java.quickstart.hw1.car;
+
+public class CarService {
+    private Car car;
+
+    boolean result;
+
+    public CarService(Car car) {
+        this.car = car;
+    }
+
+    public Car getCar() {
+        return car;
+    }
+
+    public void setCar(Car car) {
+        this.car = car;
+    }
+
+
+    public void accelerate(int targetSpeed) {
+        while (car.getCurentSpeed() < targetSpeed) {
+            double newSpeed = car.getCurentSpeed() + 1;
+            if (newSpeed > car.getMaxSpeed()) {
+                newSpeed = car.getCurentSpeed();
+            }
+            car.setCurentSpeed(newSpeed);
+            if (targetSpeed > car.getMaxSpeed()) {
+                newSpeed = car.getMaxSpeed();

```

Comment:
this looks like check that could be encapsulate into method with meaningful name